src/UserSettingsPage.cpp

```diff
@@ -499,18 +436,20 @@ UserSettingsPage::showEvent(QShowEvent *)
 void
 UserSettingsPage::resizeEvent(QResizeEvent *event)
 {
-        mainLayout_->setContentsMargins(0, LayoutTopMargin, 0, LayoutBottomMargin);
-        double contentMinWidth = mainLayout_->minimumSize().width();
+        auto preWidth_ = width();
+
+        formLayout_->setContentsMargins(0, LayoutTopMargin, 0, LayoutBottomMargin);
+        double contentMinWidth = formLayout_->minimumSize().width();
```

Comment:
We can replace these two lines with just:

```suggestion
        double contentMinWidth = deviceFingerprintValue_->width();
```

Reply:
this feels a bit fragile, if another wide field gets added later or the rendering of the value changes down the road it could make for a frustrating bug to chase down.

Reply:
Probably, but it makes the double layouting go away, that makes resizing the widow currently a lot more expensive, since it needs to calculate the sizes of every widget twice it seems And it only means, that it may reduce the margins a bit too late, which is already an edge case.

Reply:
well in that case, maybe we add another static value to the header for a minimum content width, so we can at least have a knob to tweak. and we can add a comment saying its currently constrained by the fingerprint value.